collector.go

```diff
@@ -111,8 +114,20 @@ func (c *Collector) Collect(ctx context.Context) error {
 				return microerror.Mask(err)
 			}
 
-			c.logger.Log("endpoint", e, "level", "debug", "message", fmt.Sprintf("collector found %d version bundles from endpoint", len(r.VersionBundles)))
-			bundles = append(bundles, r.VersionBundles...)
+			var filteredBundles []Bundle
+
+			if c.filterFunc != nil {
+				for _, b := range r.VersionBundles {
+					if c.filterFunc(b) {
+						filteredBundles = append(filteredBundles, b)
```

Comment:
Logging for the stuff we filter would be nice.

Reply:
You mean logging each bundle that was filtered out (and not included)?

Reply:
Yes.